packages/create/src/gather-user-responses.ts

```diff
@@ -80,6 +84,18 @@ export async function gatherUserResponses(root: string): Promise<UserResponses>
                 active: 'yes',
                 inactive: 'no',
             },
+            {
+                type: 'text',
+                name: 'superadminIdentifier',
+                message: 'What identifier do you want to use for the superadmin user?',
+                initial: SUPER_ADMIN_USER_IDENTIFIER,
+            },
+            {
+                type: 'text',
+                name: 'superadminPassword',
+                message: 'WWhat password do you want to use for the superadmin user?',
```

Comment:
Typo "WWhat"

Reply:
Thanks, good catch! I'll update the pull request.